source/js/raffler.js

```diff
@@ -36,7 +36,13 @@ var Raffler = {
     onKeyDown: function(e) {
         // We only handle the space (32) and page down (34) keys. Page down is enabled because
         // presentation remotes emit page down on the "next" button
-        if (e.keyCode != 32 && e.keyCode != 34) {
+        if (e.keyCode != 32 && e.keyCode != 34 && e.keyCode != 27) {
```

Comment:
Although it's unrelated do you think that would be nice to use `KeyboardEvent.key` instead of `KeyboardEvent.keyCode`? `KeyboardEvent.keyCode` is [deprecated according to MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)